config/armada/config.yaml

```diff
@@ -49,6 +49,11 @@ scheduling:
   maxPodSpecSizeBytes: 65535
   minJobResources:
     memory: 1Mi
+  preemption:
+    enabled: false
+  minTerminationGracePeriod: 5s
+  maxTerminationGracePeriod: 300s
+  defaultTerminationGracePeriod: 60s
```

Comment:
Let's make the default equal to the minimum, i.e., 5s, for now. Since the current behaviour is 0s, I'd prefer to use a low default initially.

Reply:
will do